conn/zstrm.c

```diff
@@ -105,9 +105,9 @@ static int zstrm_close(struct Connection *conn)
 
   mutt_debug(LL_DEBUG5, "read %lu->%lu (%.1fx) wrote %lu<-%lu (%.1fx)\n",
              zctx->read.z.total_in, zctx->read.z.total_out,
-             (float) zctx->read.z.total_out / (float) zctx->read.z.total_in,
+             (double) zctx->read.z.total_out / (double) zctx->read.z.total_in,
```

Comment:
i think the second cast can be omitted, as the second operand will be promoted automatically to match.